wp-battlenet-plugin.php

```diff
@@ -7,4 +7,11 @@
  */
 add_action('init', function () {
     error_log('Hello World');
-});
\ No newline at end of file
+});
+
+function say_something_func(){
+    return '<h1>HELLO I AM SAYING SOMETHING</h1>';
+}
+add_shortcode('say_something', say_something_func());
```

Comment:
```
Notice: Function do_shortcode_tag was called incorrectly. Attempting to parse a shortcode without a valid callback: say_something Please see Debugging in WordPress for more information. (This message was added in version 4.3.0.) in /var/www/html/wp-includes/functions.php on line 6078
```

This error is because you are invoking the function immediately. How you have it currently you aren't passing `add_shortcode` a callback. You are passing it the return of  `say_something_func`

```suggestion
add_shortcode('say_something', 'say_something_func');
```